F1WM/Controllers/NewsController.cs

```diff
@@ -96,6 +96,18 @@ public async Task<ActionResult<IEnumerable<NewsSummary>>> GetRelatedNews(int new
 			var news = await service.GetRelatedNews(newsId, before, count);
 			return this.NotFoundResultIfNull(news);
 		}
+		
+		[HttpGet("search/{term}")]
+		[ProducesResponseType(200)]
+		[ProducesResponseType(404)]
+		public async Task<PagedResult<NewsSummary>> SearchNews(string term,
```

Comment:
It might be confusing that order of parameters is changed from `term, before, page, countPerPage` in controller call to `term, page, countPerPage, before` in service call.